video2dataset/workers/optical_flow_worker.py

```diff
@@ -167,11 +104,19 @@ def process_shard(
         shard, shard_id = row
         start_time = time.time()
 
-        fs, shard_path = fsspec.core.url_to_fs(shard[: -len(".tar")] + ".parquet")
-
-        with fs.open(shard_path, "rb") as f:
-            df = pa.parquet.read_table(f)
-            schema = df.schema
+        try:
+            fs, shard_path = fsspec.core.url_to_fs(shard[: -len(".tar")] + ".parquet")
+
+            with fs.open(shard_path, "rb") as f:
+                df = pa.parquet.read_table(f)
+                schema = df.schema
+        except Exception as e:  # pylint: disable=broad-except,unused-variable
+            fields = [
```

Comment:
does this mean your other PR is not needed anymore?

Reply:
it is not needed -- it has other additions like batching RAFT, but I think we're delaying that until later so it's not really necessary rn. we can close the PR